[Bug]: proot fails to build

### Problem description

proot fails to build.
Here are the logs:

GEN	build.h
1 warning generated.
  LD	loader/loader-m32
  CC	cli/cli.o
  CC	cli/proot.o
  CC	execve/aoxp.o
  CC	syscall/seccomp.o
  CC	path/path.o
  CC	tracee/mem.o
  CC	extension/extension.o
readelf -s loader/loader | awk -f loader/loader-info.awk > loader/loader-info.c
  CC	loader/loader-info.o
./cli/cli.c:572:14: warning: unused variable 'pointers' [-Wunused-variable]
  572 |         void *const pointers[] = { this_function, call_site };
      |                     ^~~~~~~~
2 warnings generated.
1 warning generated.
  LD	proot
rm .check_seccomp_filter.o .check_process_vm.o .check_seccomp_filter .check_process_vm
make: Leaving directory '/home/builder/.termux-build/proot/src/src'
make: Entering directory '/home/builder/.termux-build/proot/src/src'
  INSTALL	proot
install -D loader/loader /data/data/com.example.app/files/usr/libexec/proot/loader
install -D loader/loader-m32 /data/data/com.example.app/files/usr/libexec/proot/loader32
make: Leaving directory '/home/builder/.termux-build/proot/src/src'
llvm-strip: error: './libexec/proot/loader': failed to allocate memory buffer of 1fffe006d8 bytes


### What steps will reproduce the bug?

build-package.a all proot

### What is the expected behavior?

_No response_

### System information

```shell
Ubuntu
```

--- a/packages/proot/0001-link-rosegment.patch
+++ b/packages/proot/0001-link-rosegment.patch
@@ -1,0 +1,11 @@
+--- a/src/GNUmakefile
++++ b/src/GNUmakefile
+@@ -206,7 +206,7 @@
+ $(eval $(call define_from_arch.h,$1,LOADER_ADDRESS))
+ 
+ LOADER_CFLAGS$1  += -fPIC -ffreestanding $(LOADER_ARCH_CFLAGS$1)
+-LOADER_LDFLAGS$1 += -static -nostdlib -Wl$(BUILD_ID_NONE),-Ttext=$(LOADER_ADDRESS$1),-z,noexecstack
++LOADER_LDFLAGS$1 += -static -nostdlib -Wl$(BUILD_ID_NONE),-Ttext=$(LOADER_ADDRESS$1),--rosegment,-z,noexecstack
+ 
+ loader/loader$1.o: loader/loader.c
+ 	@mkdir -p $$(dir $$@)

--- a/packages/proot/build.sh
+++ b/packages/proot/build.sh
@@ -6,7 +6,7 @@ TERMUX_PKG_MAINTAINER="Michal Bednarski @michalbednarski"
 # Just bump commit and version when needed:
 _COMMIT=228a5f28b078f4e2504de46758ce17948f73f507
 TERMUX_PKG_VERSION=5.1.107
-TERMUX_PKG_REVISION=67
+TERMUX_PKG_REVISION=68
 TERMUX_PKG_SRCURL=https://github.com/termux/proot/archive/${_COMMIT}.zip
 TERMUX_PKG_SHA256=39ed5eeb6ebde37be8e53591c670bf35d05869f196eff697c8dd1c20af3c0034
 TERMUX_PKG_AUTO_UPDATE=false

